src/main/scala/junctions/crossing.scala

```diff
@@ -91,3 +90,46 @@ object ClockToSignal {
     c2s.io.signal_out
   }
 }
+
+object LevelCrossing {
+  class SynchronizerBackend[T <: Data](typ: T, sync: Int, _clock: Clock) extends Module(Some(_clock)) {
```

Comment:
Hmm, yes I see that could be annoying. Or if you had a UInt of width 1 it would also yell at you even though it's technically fine.